app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/balance/BalancePopUpScreen.kt

```diff
@@ -283,33 +290,40 @@ fun DisplayEditBalance(balanceDetailedViewModel: BalanceDetailedViewModel) {
             item {
               Row(
                   modifier = Modifier.fillMaxWidth().padding(8.dp),
-                  horizontalArrangement = Arrangement.SpaceBetween,
+                  horizontalArrangement = Arrangement.End,
               ) {
-                Button(
-                    onClick = { balanceDetailedViewModel.deleteBalanceItem(balance.uid) },
-                    modifier = Modifier.testTag("editDeleteButton"),
-                ) {
-                  Text("Delete")
+                if (balanceModel.editing) {
+                  TextButton(
+                      content = { Text("Delete") },
```

Comment:
```suggestion
                      content = { Text("Delete",color = MaterialTheme.colorScheme.error) },
```

Make the delete text red, maybe?